script.skinshortcuts/addon.xml

```diff
@@ -1,41 +1,64 @@
-		<description lang="pl_PL">Wtyczka dla skór, która zapewnia proste zarządzanie i tworzenie list skrótów użytkownika (wymaga wsparcia skóry)</description>
-		<description lang="pt_PT">Um add-on que permite que os temas tenham recursos simples para gerir e listar atalhos de utilizadores (é necessário suporte do tema)</description>
-		<description lang="pt_BR">Add-on para que as skins possam prover gerenciamento de lista de atalhos para o menu principal (requer suporte da skin)</description>
-		<description lang="ru_RU">Дополнение для обложек, предоставляющее простой способ управления и список пользовательских ярлыков (требуется поддержка обложкой)</description>
-		<description lang="sk_SK">Doplnok poskytujúci jednoduchý manažment zoznamu užívateľských odkazov (vyžaduje podporu v doplnku pre vzhľad)</description>
-		<description lang="sv_SE">Skinntillägg som tillhandahåller enkel hantering och översikt av användargenvägar (om skinnet stöder det)</description>
-		<description lang="zh_TW">插件为皮肤提供了简单的管理和用户连接方式列表 (需要皮肤支持)</description>
-		<platform>all</platform>
-		<license>GPL-2.0-only</license>
-		<forum>https://forum.kodi.tv/showthread.php?tid=178294</forum>
-		<source>https://github.com/mikesilvo164/script.skinshortcuts</source>
-		<assets>
-			<icon>icon.png</icon>
-			<fanart>fanart.jpg</fanart>
-		</assets>
-		<news>Kodi v19 Matrix + Python 3 compatibility</news>
-	</extension>
+<addon id="script.skinshortcuts" name="Skin Shortcuts" version="2.0.0" provider-name="BigNoid, marcelveldt, SiLVO">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.unidecode" version="1.1.1+matrix.2"/>
+        <import addon="script.module.simpleeval" version="0.9.10"/>
+    </requires>
+    <extension point="xbmc.python.library" library="resources/lib/entry_point.py"/>
+    <extension point="xbmc.addon.metadata">
+        <news>
+- restructured (no change to API)
+- added copyright and spdx license headers
+- housekeeping
+    - cleaned up whitespace
+    - unused variables
+    - unused imports
+    - renamed shadowed builtins
+    - applied PEP8
+    - linted with pylint, isort
+    - updated/removed po file comments and metadata
+- moved shared/common constants to their own module
+- refactored json rpc requests and moved to a common module
+- refactored logging and moved to a common module
+- refactored hash generation and moved to a common module
+- refactored file handling and moved to a common module
+- removed any unnecessary string encoding/decoding
+- removed duplicate code
+- assorted minor fixes
+- updated settings.xml to Kodi's new settings format
+- added to Weblate for translations
+- added ::ENABLE:: to enable an add-on, used like ::INSTALL::
+        </news>
+        <platform>all</platform>
+        <license>GPL-2.0-only</license>
+        <forum>https://forum.kodi.tv/showthread.php?tid=178294</forum>
+        <source>https://github.com/mikesilvo164/script.skinshortcuts</source>
```

Comment:
Still valid?

Reply:
Still valid, will be maintained at the same repo